src/libs/LocaleDigitUtils.ts

```diff
@@ -13,28 +13,31 @@ const INDEX_DECIMAL = 10;
 const INDEX_MINUS_SIGN = 11;
 const INDEX_GROUP = 12;
 
-const getLocaleDigits = _.memoize((locale: Locale): string[] => {
-    const localeDigits = [...STANDARD_DIGITS];
-    for (let i = 0; i <= 9; i++) {
-        localeDigits[i] = NumberFormatUtils.format(locale, i);
-    }
-    NumberFormatUtils.formatToParts(locale, 1000000.5).forEach((part) => {
-        switch (part.type) {
-            case 'decimal':
-                localeDigits[INDEX_DECIMAL] = part.value;
-                break;
-            case 'minusSign':
-                localeDigits[INDEX_MINUS_SIGN] = part.value;
-                break;
-            case 'group':
-                localeDigits[INDEX_GROUP] = part.value;
-                break;
-            default:
-                break;
+const getLocaleDigits = memoize(
+    (locale: Locale): string[] => {
+        const localeDigits = [...STANDARD_DIGITS];
+        for (let i = 0; i <= 9; i++) {
+            localeDigits[i] = NumberFormatUtils.format(locale, i);
         }
-    });
-    return localeDigits;
-});
+        NumberFormatUtils.formatToParts(locale, 1000000.5).forEach((part) => {
```

Comment:
the `1000000.5` looks like random magic number, can we extract it into a variable?

Reply:
I agree, but I am not sure of the of the this constant. Do you know what is the history behind that? @gedu

Reply:
No idea xD, but maybe something like: sampleFormattedNumber or localeSampleNumber.

Reply:
Yeah in that case I would leave it as-is and find someone who knows the context :D It was in the codebase in this form and it is outside the scope of the PR (IKR it's just a simple number and word 'scope' might be too much xD), so it should not harm us. WDYT?